rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1.json

```diff
@@ -1591,6 +1591,103 @@
         "minLength": 4
       }
     },
+    "editionStatement": {
+      "title": "Edition Statement",
```

Comment:
can you ask the PO for title and description for all the new fields?

Reply:
Ok with the PO: "Edition statement"

Reply:
Ok with PO: Edition statement